### DISCUSSION TITLE
Mongolia carrier network settings

### DISCUSSION THREAD

**Discussion by Ondo-Dulguun** [kyujin-cho/pixel-volte-patch#249] [Carrier-specific Experiences]
Hi,

How can I to add unsupported carrier settings into PixelIMS and Shizuku?

**Comment by fillwithjoy1** [comment]
Could you please elaborate on what your question is about.

**Reply by Ondo-Dulguun** [reply]
1.MNC=428, MCC=33 and IMS, ePDG settings support?
2.Pixel 8 Pro can't register to our IMS, only send once REGISTER request and then stuck...

**Reply by fillwithjoy1** [reply]
I don't think I can solve your issue, but I'll get @kyujin-cho to look at this

**Reply by Ondo-Dulguun** [reply]
Yes, please

Thanks

**Comment by Puujee0238** [comment]
@Ondo-Dulguun Hi, Ene deer shiidel olsnu hoo?

**Reply by Ondo-Dulguun** [reply]
LTE modem software issue bolson, zasahiig huleej bn

**Reply by Puujee0238** [reply]
Google pixel deer hediigeer shiidegdeh bh gsn baragtsaa hugataa bii yu hoo?

**Reply by Ondo-Dulguun** [reply]
2024 on dotor gesen...
